packages/karma-build-scripts/LICENSE

```diff
@@ -0,0 +1,192 @@
+   MODIFIED VERSION OF APACHE LICENSE, VERSION 2.0
```

Comment:
IMHO you should not make any confusing references to the Apache license in name anywhere in your license text. Call this whatever you want but not a modified Apache 2.0 which brings confusion